### DISCUSSION TITLE
[AVM-PA] freetz 7590 wlan not working any longer

### DISCUSSION THREAD

**Discussion by jpsollie** [Freetz-NG/freetz-ng#170] [Allgemein]
after installing freetz, my wlan did not work anymore,
there must be an error in the config which causes the wlan setup to fail.
resetting the default options (so minimal modifications) restored wlan functionality
[working_config.txt](https://github.com/Freetz-NG/freetz-ng/files/5864894/working_config.txt)
[no_wlan_config.txt](https://github.com/Freetz-NG/freetz-ng/files/5864896/no_wlan_config.txt)

non-working .config + working .config attached,
(image file available on request)
Could anyone point me to a *possible* strategy on how to bisect the issue?

**Comment by kainers1** [comment]
In any case, it would be helpful to see the config.compressed because it is more lightweight.

As far as I have seen, you have enabled some packages and removal patches. Try to deactivate this selection step by step (It can also be several steps at once) and test if wifi works again.

**Comment by jpsollie** [comment]
any idea of *possible* targets I should focus on first?

**Comment by kainers1** [comment]
I would start as follows: As step 1 the removal patches and as step 2 Wireguard and Openvpn.

**Comment by fda77** [comment]
Maybe the configuration of the VPN packages messed with the network-interfaces or ip-ranges/routing. You coudl try to disable them in webif and reboot

**Comment by fda77** [comment]
You should not use
```
FREETZ_USER_LEVEL_DEVELOPER
```
which enables things which my not are working yet, like iptables:
```
FREETZ_MODULES_OWN="ah6 esp6 ip6_tunnel sit tunnel6 ip_tables iptable_filter xfrm6_mode_transport x_tables"
```
So it seems you've also altered the kernel-.config

**Reply by jpsollie** [reply]
Unless that's not the way to do it, I did it because I want to enable ipv6 tunneling to work
As such, I thought I needed ah6, esp6, and iptables to organize routing tables.
I intend to try to install strongswan as well, as avm's multid daemon seems pretty bad @ routing  ipv6

*EDIT*: I made a new config with all required modules from strongswan's page (and assumed I could live with only esp6 ah6 and ipv6_tunnel as modules, so no xfrm or iptables), and this seems the way to go! let's see whether I could manage creating a strongswan setup now :)

**Comment by fda77** [comment]
AVM patched the kernel to make its "Packet Accelerator" to work. The PA routes packes by hardware and not by the kernel! So some things of iptables dont work, dont compile or even cause crashes. Thats the reaseon why it is for "developer" only
